src/string_tensor_pack.hpp

```diff
@@ -30,12 +36,6 @@ class StringTensorPack : public ov::op::Op {
         return true;
     }
 
-    bool has_evaluate() const override {
-        return true;
-    }
-
-    bool evaluate(ov::TensorVector& outputs, const ov::TensorVector& inputs) const override;
-
```

Comment:
Since this is deprecated, let us not change this code. No reason to optimize it, etc.
Less changes - less bugs.